miner/src/pool/scoring.rs

```diff
@@ -67,21 +67,21 @@ impl NonceAndGasPrice {
 	}
 }
 
-impl txpool::Scoring<VerifiedTransaction> for NonceAndGasPrice {
+impl<P: ?Sized> txpool::Scoring<P> for NonceAndGasPrice where P: ScoredTransaction + txpool::VerifiedTransaction + Sized {
```

Comment:
So is it `?Sized` or `Sized`? I don't think we need either of the bounds (just rely on default `Sized`)

Reply:
Removed